### ISSUE TITLE
Importing the same container multiple times doesn't behave the same as in docker

### ISSUE THREAD

**Issue by romainreuillon** [indigo-dc/udocker#45]
```
[reuillon:~/Install/udocker] devel 34m53s ± udocker import /tmp/httptest.tar youpi
Error: tag already exists in repo:  latest
Error: importing file
[reuillon:~/Install/udocker] devel 34m56s 1 ± docker import /tmp/httptest.tar youpi
sha256:2ab8ff990b73e9a5ff10d06c2ae809ea7dc1f1faffc84c8dbdbffa6f9877c855
[reuillon:~/Install/udocker] devel 35m2s ± docker import /tmp/httptest.tar youpi
sha256:13a6a8a6718c86bd5222bf311fa4be7d19f33f14d38abf3d76e5452bc1996675
```

**Comment by jorge-lip** [comment]
Same issue as #46  
Since docker uses IDs it can move the name to the newly imported layer leaving the older with just the ID and no name/tag. For now udocker cannot do this as the name is the ID.

**Comment by mariojmdavid** [comment]
solved see comment on #46 

latest pre release version 1.2.9 from devel3 this is solved

  udocker import ub.tar ubuntu
  Info: added layer cdf9b77bcfed92625077f67af33e63517bfc0d6985451a6c15dd7b369c7bb936

  udocker images
  REPOSITORY
  ubuntu:latest

Also

  udocker import ub.tar ubuntu:20.04
  Info: added layer 8c8134e42a253f3b5891e0614ce57a07b2c810cdc5fd7d85a088c6c585c1f2d3
  udocker images
  REPOSITORY
  ubuntu:20.04                                                 .
  
thus this is solved and closed
